src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,140 @@
+---
+title: Inclusive Making and Hacking
+layout: default
+category: Methods
+---
+
+Hackathons, makeathons and similar events have become very popular in recent years, both in traditional learning contexts such as schools, colleges and universities, and in broader world.
+
+Ideas of "hacking" and "making" have also been popularized as approaches to creatively and collaboratively solve problems in society, business and government.
+
+Where does Inclusive Design fit into these approaches?
+
+## On Terminology
+
+### Hackathon
+
+We use the term "hackathon" expansively in this entry to refer to events that emphasize intensive short-term collaboration between small groups, usually (but not always) physically co-located. The groups typically ideate, design and prototype around themes such as <a href="http://arthackathon.co.uk/" target="_blank" class="link-external">art</a> or <a href="http://civictech.ca/2016/04/08/codeacross-toronto-community-collaboration-in-celebration-of-international-open-data-day/" target="_blank" class="link-external">open data</a>, then present their result in some form to to the larger gathering throughout the event or at the end.
```

Comment:
"then present their result in some form to to the larger gathering throughout the event or at the end"

Double "to".